### ISSUE TITLE
Migrating existing projects "into" the SAFE template

### ISSUE THREAD

**Issue by isaacabraham** [SAFE-Stack/SAFE-template#230]
I've heard from a few people that are starting out with SAFE apps that they are coming up against the challenge of already having created some application, but would like to "move" to the SAFE template. The question is, how best to do this e.g. documentation? some tooling? related to the "componentisation" effort of SAFE (with nuget packages etc.)?

I've created this issue just as a mean to track this and encourage some discussion on it e.g. what the user experience and challenges are, and how we could best solve them.

cc: @forki @theimowski @alfonsogarciacaro @Krzysztof-Cieslak @bruinbrown

**Comment by theimowski** [comment]
Yes that's somehow related to what I wanted to achieve with https://github.com/SAFE-Stack/SAFE-template/issues/103

As of now I don't have a better idea than creating SAFE-specific NuGet packages, be it meta-packages with pinned dependencies, or including some logic e.g. for deploy tasks in build scripts

We could and should probably extend that to JS as well - i.e. SAFE npm packages with pinned dependencies
